doc/source/whatsnew/v2.0.0.rst

```diff
@@ -146,6 +146,14 @@ Deprecations
 
 Removal of prior version deprecations/changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- Removed deprecated :meth:`Categorical.to_dense`, use ``np.asarray(cat)`` instead (:issue:`32639`)
+- Removed deprecated :meth:`Categorical.take_nd` (:issue:`27745`)
+- Removed deprecated :meth:`Categorical.mode`, use ``Series(cat).mode()`` instead (:issue:`45033`)
```

Comment:
@jbrockmendel I don't think this was actually removed in this PR?

Reply:
good catch, will make a PR now